app/api/auth.py

```diff
@@ -1,27 +1,31 @@
+import os
 import base64
 import random
 import string
 
 import requests
-from flask import request, jsonify, make_response, Blueprint
+from flask import request, jsonify, make_response, Blueprint, send_file, redirect
 from flask_jwt import current_identity as current_user, jwt_required
 from sqlalchemy.orm.exc import NoResultFound
 
 from app import get_settings
 from app.api.helpers.db import save_to_db, get_count
-from app.api.helpers.errors import UnprocessableEntityError, NotFoundError, BadRequestError
+from app.api.helpers.errors import ForbiddenError, UnprocessableEntityError, NotFoundError, BadRequestError
 from app.api.helpers.files import make_frontend_url
 from app.api.helpers.mail import send_email_with_action, \
     send_email_confirmation
 from app.api.helpers.notification import send_notification_with_action
 from app.api.helpers.third_party_auth import GoogleOAuth, FbOAuth, TwitterOAuth, InstagramOAuth
 from app.api.helpers.utilities import get_serializer, str_generator
 from app.models import db
+from app.models.order import Order
 from app.models.mail import PASSWORD_RESET, PASSWORD_CHANGE, \
     USER_REGISTER_WITH_PASSWORD, PASSWORD_RESET_AND_VERIFY
 from app.models.notification import PASSWORD_CHANGE as PASSWORD_CHANGE_NOTIF
 from app.models.user import User
 
+
+ticket_blueprint = Blueprint('ticket_blueprint', __name__, url_prefix='/static/media/attendees/tickets/pdf')
```

Comment:
This shouldn't be needed now

Reply:
The tickets are still being stored at that location. Do you want me to alter it so that it gets saved in a dir `<order_identifier>/tickets` ?

Reply:
Will Update the PR with requested changes.

Reply:
They shouldn't be stored in static folder

Reply:
@iamareebjamal Moved them to a temp folder. We can schedule the deletions later. Additionally, removed generic exception handling.